addons/mrp_subcontracting/wizard/change_production_qty.py

```diff
@@ -11,3 +11,12 @@ class ChangeProductionQty(models.TransientModel):
     def _need_quantity_propagation(self, move, qty):
         res = super()._need_quantity_propagation(move, qty)
         return res and not any(m.is_subcontract for m in move.move_dest_ids)
+
+    @api.model
+    def _update_product_qty(self, move, qty):
```

Comment:
can be done in stock.move `write()` method

Reply:
As discussed, it might raise issues, so I'll let it like that for now.